src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -37,7 +40,12 @@ async fn run() {
             return;
         }
 
-        let result = ClientOptions::parse(&test_file.uri).await;
+        let result = if cfg!(target_os = "windows") {
+            ClientOptions::parse_with_resolver_config(&test_file.uri, ResolverConfig::cloudflare())
```

Comment:
I thought these tests relied on the DNS records set on the system, meaning using the cloudflare config should cause it to fail. They seem to be passing though, which makes me concerned for my implementation of the config.

Reply:
I'm pretty sure that putting something in /etc/hosts (or whatever the equivalent is on Windows) will still resolve even if you don't use the system resolver. Either way though, these records do exist; see [an SRV lookup of one of the domains on some random DNS lookup site I found when googling](https://dnschecker.org/all-dns-records-of-domain.php?query=_mongodb._tcp.test5.test.build.10gen.cc&rtype=SRV). You might be thinking of the `ldaptest.10gen.cc` records, which I think are not published.

Reply:
Oh wow, okay, so the SRV records do resolve. I wonder if the "localhost.test.build.10gen.cc" will too, though. I guess we'll find out in RUST-489.